src/python/pants/notes/1.17.x.rst

```diff
@@ -82,7 +82,7 @@ Bugfixes
 * Fix dependency cycle between `backend/native/subsystems` and `backend/python/subsystems` (#7793)
   `PR #7793 <https://github.com/pantsbuild/pants/pull/7793>`_
 
-* Fix `create_subprocess_encoding_environment` rule. (#7789)
```

Comment:
Should probably not find-replace into notes.

Reply:
Agreed, missed this. Thanks!